support/packages/github-to-jira-synchronization-tool/src/jira/JiraRetryHandler.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const isJiraUp = require('../utils/isJiraUp');
+
+module.exports = class JiraRetryHandler {
```

Comment:
We need a RetryHandler for JIRA :scream:? ... :roll_eyes:

Reply:
Yeah, in case jira is down for some time we don't want to lose those events :'( historically it has been down just a couple of times, so it doesn't seems like a big deal

Reply:
No, no, I like it. I found it a very good idea :+1: 

As long as the user gets immediate feedback if he changed the config and the load went wrong it's good. Otherwise (if the user won't be clearly notified) it may be a cause of trouble if the user looks at the config and think it's right but the application does a different thing.

Reply:
Yes, we all know JIRA :facepalm: